apex/models/block_storage_model.go

```diff
@@ -225,3 +225,12 @@ func GetBlockStorageSystem(storageSystem client.StorageSystemsInstance) (blockSt
 
 	return blockStorageState
 }
+
+// ConvertTimeToString converts time to string
+func ConvertTimeToString(pTime *time.Time) string {
```

Comment:
@doriac11 Can we move to general helper? May be other places can use this generic method too.

Reply:
@nikitajoshi1, this is only used here.
It was in the general helper, but I moved it here since it causes a circular dependency provider -> helper -> model -> helper.

Reply:
@nikitajoshi1 @shekhar-j , I have made the change to move this back to general_helper, and moved the tf generator GetBlockStorageSystem to helper so we dont have the circular dependency.